flooding/flooding.proto

```diff
@@ -2,7 +2,7 @@ package floodsub.pb;
 
 message RPC {
 	repeated SubOpts subscriptions = 1;
-	repeated Message publish = 2;
```

Comment:
maybe `publishMsgs` ? the point is to be clear that these messages are for publishing